src/main/java/dk/kb/present/StorageHandler.java

```diff
@@ -14,22 +14,19 @@
  */
 package dk.kb.present;
 
+import dk.kb.present.storage.DSStorage;
 import dk.kb.present.storage.Storage;
 import dk.kb.present.storage.StorageController;
 import dk.kb.util.yaml.YAML;
-import javassist.NotFoundException;
```

Comment:
What was that and how did it ever work!?

Reply:
wauw! No clue!